app/views/works/index.html.erb

```diff
@@ -0,0 +1,74 @@
+<main>
+  <h2>List of Works</h2>
+  <section class="media-table">
+    <h4>Albums</h4>
+    <table class="table">
+      <tr>
+        <th scope="col">Votes</th>
+        <th scope="col">Title</th>
+        <th scope="col">Created By</th>
+        <th scope="col">Published</th>
+        <th scope="col">Upvote</th>
+      </tr>
+      <% Work.ordered_filter("album").each do |work| %>
+        <% if work %>
+          <tr>
+            <th scope="row"><%= work.votes.length %></th>
+            <td><%= link_to work.title, work_path(work.id) %></td>
+            <td><%= work.creator %></td>
+            <td><%= work.publication_year.year %></td>
+            <td><%= button_to "Upvote", upvote_work_path(work.id), class: "btn btn-primary" %></td>
+          </tr>
+        <% end %>
+      <% end %>
```

Comment:
Note that this code is essentially repeated 3 times. Consider how you could DRY this up with a loop or a partial view.